Builder/FormContractor.php

```diff
@@ -22,6 +22,11 @@ class FormContractor implements FormContractorInterface
     protected $fieldFactory;
 
     /**
+     * @var FormFactoryInterface
+     */
+    protected $formFactory;
```

Comment:
On the other hand, it's accessible through the getter... so no reason to not change it. I'll update the PR.